### PR TITLE
Add post about layout mobile-first re-design

### PR DESCRIPTION
**Pull request by saraycp**
This is the first blog post of a series regarding mobile-first redesign. This time describing the layout redesign.

### CODE REVIEW

**Comment by hennevogel** [comment]
Deploy preview for *openbuildservice* ready!

Built with commit cf5d0a82bacd248260370e25884724362e83df4f

https://deploy-preview-242--openbuildservice.netlify.com

**Comment by saraycp** [comment]
> I'm never a fan of having emojis in a blog post, but it's not a blocker.

well, there are many on the top. I'll remove some of them!

File: _posts/development/2020-02-17-mobile-first-layout.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+Do you like checking the status of your OBS account on the way to work? While sitting on the sofa? On a flight?
+
+You are not alone :raised_hands:. Many of the OBS users like that, but the OBS web interface is often a pain in the neck, especially on
+small screen sizes.
+
+We want to change this. And with the new UI technology we introduced last year, we have the chance to do so! :clap:
+So in the last couple of weeks, we have focused on improving the user experience following a mobile-first approach
+(start the design of the page on a small screen, which has more restrictions, then expand the page features to create a tablet or desktop version).
+
+<i class="mobile alternate icon"></i>
```
Comment (hennevogel) [inline _posts/development/2020-02-17-mobile-first-layout.md]: That icon does not display

Comment (hennevogel) [reply _posts/development/2020-02-17-mobile-first-layout.md]: I think I have a better image. Wait a minute

Comment (hennevogel) [reply _posts/development/2020-02-17-mobile-first-layout.md]: pushed

Comment (saraycp) [reply _posts/development/2020-02-17-mobile-first-layout.md]: Good idea! Now I think it can be merged

**Pull request merged**